wgpu-types/src/lib.rs

```diff
@@ -1262,6 +1478,33 @@ pub struct SwapChainDescriptor {
     pub present_mode: PresentMode,
 }
 
+impl SwapChainDescriptor {
+    pub fn new(width: u32, height: u32) -> Self {
+        Self {
+            usage: TextureUsage::OUTPUT_ATTACHMENT,
+            format: TextureFormat::Bgra8UnormSrgb,
```

Comment:
This default value is ok?

Reply:
At least [here](https://github.com/gfx-rs/wgpu-rs/blob/ffa9199402e1436ba525007754e161b08ab12bcc/examples/framework.rs#L201) it uses `TextureFormat::Bgra8Unorm` but it also states `// TODO: Allow srgb unconditionally`

Reply:
I think it's ok defaulting to `Bgra8UnormSrgb` here. WASM users can always work around by using non-default, and we want to fix this in WASM too